q2_minimap2/tests/test_minimap2_build.py

```diff
@@ -35,7 +42,7 @@ def test_build_index_subprocess_failure(self):
 
             # Calling the method under test
             with self.assertRaises(Exception) as context:
-                self.plugin.methods["build_index"](self.genome)
+                build_index(self.genome)
```

Comment:
Similar to the last PR, here you can you `self.assertRaisesRegex` to test whether the error message is as expected.